File: Python/basics/salary_calc.py
# coding=utf8
#Entrada
sal_hora = float (input("Quanto você ganha por hora? "))
horas_trab = int (input("Quantas horas você trabalhou neste mês? "))

#Processamento
salario = sal_hora * horas_trab

#Saída
print("Neste mês você vai receber R$ {0:.2f}".format(salario))

File: Java/basics/salary_calc.java
package br.com.geekuniversity.secao03;

import java.util.Scanner;

public class Exercicio06 {
	
	public static void main(String[] args) {
		
		//Declaração
		float salario_hora, salario;
		int horas_trab;
		Scanner teclado = new Scanner(System.in);
		
		//Entrada
		System.out.print("Salário/hora: ");
		salario_hora = teclado.nextFloat();
		System.out.print("Horas trabalhadas: ");
		horas_trab = teclado.nextInt();
		
		//Processamento
		salario = salario_hora * horas_trab;
		
		//Saída
		System.out.println("\nSalário Total: " + salario);
		
		teclado.close();
	
	}
}
